file/builder.go

```diff
@@ -771,9 +778,38 @@ func (b *stateBuilder) ingestRoute(r FRoute) error {
 	return b.ingestPlugins(plugins)
 }
 
+func (b *stateBuilder) getPluginSchema(entityID *string) (map[string]interface{}, error) {
```

Comment:
```suggestion
func (b *stateBuilder) getPluginSchema(pluginName string) (map[string]interface{}, error) {
```
Take a string.
This code base deals with a lot of pointers but that's not what one should do in Go usually.

Reply:
done